### PR TITLE
CANS-309. Fix Mandatory (Capybara) Accessibility Issues

### PR DESCRIPTION
**Pull request by denys-davydov**
## Description
https://osi-cwds.atlassian.net/browse/CANS-309

Things that are done within the scope of the user story:
1. Added additional yarn script to run only accessibility lint (run with: **yarn test:a11y**)
2. Refactored capybara acceptance tests to separate acceptance tests running from accessibility lint running
3. Added Accessibility Lint stage to pull-request and master jenkins jobs
4. Fixed all accessibility issues on all the CANS pages that were found by Capybara

## Tests

- [ ] I used TDD to develop the feature
- [ ] I have included unit tests
- [x] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I ran all my tests locally which were green.
- [ ] My code adds no new issues to Code Climate
- [x] I ran all the linters for the project.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: app/javascript/Application/styles/global/global-styles.sass [hunk 1]
```diff
@@ -10,4 +18,9 @@
...
   list-style-type: none
   padding: 0
-  margin: 0
\ No newline at end of file
+  margin: 0
+
+.label-text
+  color: $label
+  font-size: 1.5rem
+  font-weight: 400
```
Comment (abelayt) [inline app/javascript/Application/styles/global/global-styles.sass]: Need new line as indicated.

**Comment by denys-davydov** [comment]
retest this please

**Comment by denys-davydov** [comment]
retest this please

File: Jenkinsfile [hunk 2]
```diff
@@ -97,7 +99,18 @@ def acceptanceTestStage() {
       withDockerRegistry([credentialsId: DOCKER_REGISTRY_CREDENTIALS_ID]) {
         sh "docker-compose up -d --build"
         sh "sleep 120"
-        sh "docker-compose exec -T --env CANS_AUTHORIZATION_ENABLED=true --env CANS_WEB_BASE_URL=cans-web:3000/cans cans-test bundle exec rspec spec/acceptance"
+        sh "docker-compose exec -T cans-test bundle exec rspec spec/acceptance"
```
Comment (edgibbs) [inline Jenkinsfile]: So pulling the environment variables here means they have to exist correctly on the environment where they're run.  Are we sure this has been updated for the cans-master build?

File: spec/acceptance_helpers/resource_helper.rb [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    fill_in('Last Name', with: client['last_name'])
+    focus_and_fill_in('#dob', client['dob'])
+    find('.case-numbers-single-control').click
+    focus_and_fill_in('#caseNumber0', client['case_number_0'])
+    focus_and_fill_in('#caseNumber1', client['case_number_1'])
+    focus_and_fill_in('#external_id', client['client_id'])
+    find('#county-select').find(:xpath, 'option[11]').select_option
+  end
+  # rubocop:enable Metrics/AbcSize
+end
```
[the remark this answers is not in the document]
Comment (denys-davydov) [reply spec/acceptance_helpers/resource_helper.rb]: So the `test/helpers` folder was renamed to `test/acceptance_helpers` and the `.keep` file is still in the folder

File: Jenkinsfile (hunk 2 above)
Comment (denys-davydov) [reply Jenkinsfile]: Ok, I've added `CANS_WEB_BASE_URL` param to cans-master-build.  The second one has the same default value so there is no need to have it in the job.

File: spec/acceptance_helpers/resource_helper.rb (hunk 3 above)
[the remark this answers is not in the document]
Comment (edgibbs) [reply spec/acceptance_helpers/resource_helper.rb]: But it shouldn't have changed at all, we aren't running any Mini::Test tests, it's all RSpec in the spec/ directory.

File: Jenkinsfile (hunk 2 above)
Comment (edgibbs) [reply Jenkinsfile]: OK.

File: spec/acceptance_helpers/resource_helper.rb (hunk 3 above)
[the remark this answers is not in the document]
Comment (denys-davydov) [reply spec/acceptance_helpers/resource_helper.rb]: Could you please be more specific on the issue? What exactly is wrong here?

**Comment by edgibbs** [comment]
So looks like a lingering accessibility violation, and it needs another merge, but otherwise looks good.

**Comment by LeonidMarushevskyi** [comment]
Jenkins retest this please

**Pull request merged**